src/operations/multireweight.lisp

```diff
@@ -51,65 +51,70 @@
 ;;;
 
 (define-process-upkeep ((supervisor supervisor) now) (START-MULTIREWEIGHT pong)
-  "Sets up the multireweight procedure.
-
-1. Collect the mutually held roots for the `HOLD-CLUSTER'
-2. Lock the `HOLD-CLUSTER' and check the rootiness of each root.
-3. Change the pingability of the cluster to `:SOFT'.
-4. Scan the `HOLD-CLUSTER' for the best external rec to use for reweighting.
-5. Reweight the `HOLD-CLUSTER' according to the recommendation.
-6. Check to see if the `HOLD-CLUSTER' should be rewound, and do so if need be.
-7. Unlock the targets and tear down transient state."
+  "Sets up the multireweight procedure by first collecting mutually-held roots, which form the `HOLD-CLUSTER'."
   ;; NOTE: we couldn't call MAKE-PONG even if we wanted to, since we don't have
   ;;       access to the underlying node's Lisp object (or its type).
   (push (make-data-frame-multireweight :internal-pong nil)
         (process-data-stack supervisor))
   (with-slots (root-bucket source-root) pong
     (setf root-bucket (remove-duplicates root-bucket :test #'address=))
     (process-continuation supervisor
-                          `(CONVERGECAST-COLLECT-ROOTS ,(list source-root) ,root-bucket)
-                          `(CHECK-PRIORITY ,source-root)
-                          `(START-INNER-MULTIREWEIGHT)
+                          `(CONVERGECAST-COLLECT-ROOTS ,source-root ,root-bucket)
```

Comment:
My preference is that this still installs the whole "script" for the operation. The body of `CHECK-PRIORITY` can be wrapped by `(when aborting? ...)`, so then both commands get skipped if either of the two aborting conditions in this command are met.

Reply:
Fair, but the output of `CONVERGECAST-COLLECT-ROOTS` is used as input for `CHECK-PRIORITY` -- previously we used a data frame to deal with that but my goal here is to explicitly pass this information as an argument to `CHECK-PRIORITY'. In that case, I don't see how to install the full script at once.

Reply:
I see. I agree that that's not possible, or not easy.

There's still a data frame there, though, and CHECK-PRIORITY appears to be local to this file. What are we avoiding? Something affected by changes downstream of these?

Reply:
We're enabling something downstream, yeah

Reply:
Ah, then what can be done 🤷‍♀️